data/json/vehicleparts/wheel.json

```diff
@@ -7,15 +112,19 @@
     "location": "under",
     "symbol": "|",
     "broken_symbol": "x",
-    "difficulty": 2,
     "durability": 180,
     "description": "A strong metal wheel.",
     "damage_modifier": 70,
     "breaks_into": [ { "item": "scrap", "count": [ 4, 6 ] }, { "item": "steel_chunk", "count": [ 2, 3 ] } ],
     "rolling_resistance": 0.375,
     "wheel_type": "rigid",
     "contact_area": 80,
-    "flags": [ "WHEEL" ],
+    "requirements": {
+      "install": { "skills": [ [ "mechanics", 1 ] ], "time": "10 m", "qualities": [ { "id": "WRENCH", "level": 2 } ] },
```

Comment:
Why installing a metal wheel requires less time than installing most other wheel types? The same for removal and repairing.

Reply:
Added those values before others and forgot to update.
Metal wheel probably should be moved to obsolete items. It is uncraftable and is not found anywhere in the game (unless some hardcoded c++ location has it).